com.cocoatech.deckr.yaml

```diff
@@ -0,0 +1,121 @@
+app-id: com.cocoatech.deckr
+runtime: org.gnome.Platform
+runtime-version: "44"
+sdk: org.gnome.Sdk
+command: deckr
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-videos
+
+modules:
+  - name: deckr
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      # - mkdir -p Deckr
+      # - tar -xf Deckr-Linux-Portable.tar.gz -C Deckr
+      - cp -r Deckr /app/Deckr
+      - echo $(ls /app/Deckr)
+      - chmod +x /app/Deckr/deckr
+      - mkdir -p /app/bin
+      - ln -s /app/Deckr/deckr /app/bin/deckr
+      - mkdir -p /app/share/metainfo
+      - cp -r com.cocoatech.deckr.metainfo.xml /app/share/metainfo/
+      # - mkdir -p /app/share/icons/hicolor/scalable/apps
+      # - cp -r icon.svg /app/share/icons/hicolor/scalable/apps/com.cocoatech.deckr.svg
+      - mkdir -p /app/share/icons/hicolor/32x32/apps
+      - cp -r app_icon_32.png /app/share/icons/hicolor/32x32/apps/com.cocoatech.deckr.png
+      - mkdir -p /app/share/icons/hicolor/64x64/apps
+      - cp -r app_icon_64.png /app/share/icons/hicolor/64x64/apps/com.cocoatech.deckr.png
```

Comment:
you do not need these icon sizes, since they don't come from upstream distribution

Reply:
no idea what upstream distribution is.  

All these files were copied from your docs or other repos, so it's strange that it has so many issues. 

This has been the most confusing thing I've dealt with in 30 years.  Why not make it super simple?